Framework/PythonInterface/mantid/api/src/Exports/IPeak.cpp

```diff
@@ -128,6 +128,8 @@ void export_IPeak() {
            "Get the initial (incident) neutron energy")
       .def("getFinalEnergy", &IPeak::getFinalEnergy, arg("self"),
            "Get the final neutron energy")
+      .def("getEnergy", &IPeak::getEnergy, arg("self"),
```

Comment:
Did you consider referring to this as a delta in energy, or is there a good reason why this is just referred to as the energy?

Reply:
I did consider that, but this name was consistent with what the column in the slice viewer was called where the value was originally being calculated. I'm not sure which is better. `Energy` is a bit vague.

Reply:
I think that @AndreiSavici should provide some advice here. Deprecating parts of the python api is a difficult process, so better to get it right first time if we can.

Reply:
Not sure anyone is using that. But yes, it should be `DeltaE` or `EnergyTransfer`. You might want to change the C++ code as well

Reply:
The name in the SliceViewer should also be changed then.